### PR TITLE
Add new community page and update blog description

### PR DESCRIPTION
**Pull request by MSanKeys963**
Hi all. 👋🏻 

Added a new community page, which can be accessed at https://zarr.dev/community.

Let me know what you think; suggestions are welcome. Thanks!

Screenshot 2023-03-13 at 05 37 49

### CODE REVIEW

**Comment by MSanKeys963** [comment]
Thanks for having a look, @joshmoore.

> * I'm a bit hesitant about having "community" and "community-meetings". Before we go too far down this road, we might review the overall structure of the website (e.g. compared to other projects in pydata)

I looked at the community pages of several projects under the PyData ecosystem. 

- [NumPy](https://numpy.org/community/) and [SciPy](https://scipy.org/community/) have a page where they have listed the community meetings and different ways how a user can interact with the project and participate in the discussions.
This is already covered in community meetings, and join the community section.

- [Pandas](https://pandas.pydata.org/) doesn't have a dedicated community page; instead, they have a dropdown which expands to their blog, StackOverflow, COC, and [ecosystem](https://pandas.pydata.org/community/ecosystem.html).

- [Matplotlib](https://matplotlib.org/) doesn't seem to have a community page, or I didn't find one.

- [SciKit-Learn](https://scikit-learn.org/stable/) community page directs to their blog.

Let me know if there are any other projects I should be looking at.

After reading this, I have added a small intro text at the top here:

Screenshot 2023-03-14 at 05 40 51

I didn't mention the blog explicitly, as it can be seen and accessed from the sidebar. Let me know what you all think. Thanks!

PS. While at this, I also sent a PR to update the COC to the latest version: https://github.com/zarr-developers/.github/pull/2

**Comment by joshmoore** [comment]
> Let me know if there are any other projects I should be looking at.

None in particular. But I still find community/ and community-meetings/ a bit surprising. e.g. why not meetings as a subpage of community? It might be good to map out a sitemap for all the pages you're considering so that we can have a logical growth of the overall site.

File: community/community.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+layout: single
+author_profile: false
+title: Zarr Community
+permalink: /community/
```
Comment (joshmoore) [inline community/community.md:5]: This might not be needed if the file's name were "index.md". Do you mind giving that a try?

**Review by joshmoore — COMMENTED** [review]
Having more feedback on the site-map would still be good, but I imagine that might happen best if we get it out there. One minor comment then let's keep these things rolling.

File: community/community.md (hunk 1 above)
Comment (MSanKeys963) [reply community/community.md:5]: After removing `layout: single` and renaming the file `index.md` the sidebar disappears. I think we need this.

**Comment by MSanKeys963** [comment]
Thanks, @joshmoore. Merging this now.

**Pull request merged**